games-strategy/augustus/augustus-3.0.1.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit xdg cmake readme.gentoo-r1
+
+DESCRIPTION="Feature-enhanced open source re-implementation of Caesar III"
+HOMEPAGE="https://github.com/Keriew/augustus"
+SRC_URI="https://github.com/Keriew/augustus/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="AGPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="uncapped"
+
+DEPEND="
+	dev-libs/expat
+	media-libs/libpng:0=
+	media-libs/libsdl2[joystick,video,sound]
+	media-libs/sdl2-mixer
+"
+RDEPEND="
+	${DEPEND}
+"
+
+PATCHES="${FILESDIR}/${PN}-3.0.1-desktop_rename.patch"
```

Comment:
We normally never use a flat string for patches, it should be done similarly to julius ebuild, aka an array and patch on its own line (this generally makes it cleaner to add/remove patches as needed).

Reply:
Ok :)